### PR TITLE
Remove code and dependencies that aren't needed for BCD

### PR DESCRIPTION
**Pull request by foolip**
This isn't being developed/maintained further, and had some dependencies
on web-api-confluence-dashboard. This was found while trying to prune
dependencies as much as possible as confluence in turn has a lot of
dependencies. The dependency remains, but is now easier to understand.

### CODE REVIEW

**Comment by foolip** [comment]
@jpmedley have you been using the UI, or just the tool through the dependency in BCD? If you're only using it through BCD like I am then I'd like to simplify maintenance a bit with trimming like this.

File: scripts/build.js [hunk 1]
```diff
@@ -1,65 +0,0 @@
...
-    if (!fs.existsSync(combinedPath)) {
-      fs.mkdirSync(combinedPath);
-    }
-  }
-
-  return combinedPath;
-};
-
-// Run FOAM2 build tool
-cp.execSync(`node ${NODE_MODULES}${SEP}foam2${SEP}tools${SEP}build.js web,firebase`,
```
Comment (foolip) [inline scripts/build.js]: Removing this line is significant, making https://github.com/GoogleChromeLabs/confluence/pull/347 possible.

**Comment by foolip** [comment]
I'm done with this branch now, each commit pass the basic Travis tests and as mentioned in the last commit message, the output when used in BCD is still the same.

Barring mistakes it should be safe to merge this. https://github.com/GoogleChromeLabs/confluence/pull/347 however will require a little dance to not break BCD even temporarily. Review of both appreciated.

**Comment by foolip** [comment]
Alright, I put the "little dance" on this branch and re-confirmed that the output when used from BCD is the same. The only further reduction of dependencies possible would be by trimming deps in this repo itself, but not in this PR.

**Comment by foolip** [comment]
Thanks for review, @mdittmer! I'll merge this and make a new release, hope that works in BCD as my testing suggests it will.

**Comment by jpmedley** [comment]
I do sometimes use the UI. You are talking about the [API Catalog](http://web-confluence.appspot.com/#!/catalog), right?

**Comment by foolip** [comment]
@jpmedley, no that UI will remain and we'll keep updating with new releases. This PR removed the code supporting the cross-referencing UI with a demo at https://mdittmer.github.io/mdn-confluence/multi.html. I think you've seen it and tried it, but are you still using it?

**Pull request merged**